Makefile

```diff
@@ -1,41 +1,45 @@
 project_name = boilerPlate
 image_name = gofiber:latest
 
-run-local:
+help: ## This help dialog.
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'

```

Comment:
Can you use `grep -F` instead of `fgrep`

Reply:
[Done](https://github.com/gofiber/boilerplate/pull/103/commits/6967c2b625da1f1687c8f22bb7d864c8d2a42581)